collections/_article/godot-developer-fund.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Introducing the new Godot Development Fund"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
+ 
+
+## Reducing platform fees, amplifying impact
+
+The Development Fund will make The Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.
```

Comment:
Should we clarify what platform fees mean here? It's not just about cutting the middleman, Patreon is a platform for commercial subscriptions and thus they charge VAT. So it's not just their 5-8% cut + 5% payment processor, but an additional +20-25% VAT for EU people.

Reply:
I didn't have the numbers exactly about how much Patreon got, so I wasn't sure about adding hard numbers. If those are the percentages we could add it 100%

Reply:
Well I wouldn't try to give clear numbers either, but I think detailing that we're saving on both platform fees and, where applicable, VAT, would be useful.